_tuning-your-cluster/replication-plugin/getting-started.md

```diff
@@ -275,15 +275,15 @@ Note that you can't resume replication after it's been paused for more than 12 h
 When you no longer need to replicate an index, terminate replication from the follower cluster:
 
 ```bash
-curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:admin' 'https://localhost:9200/_plugins/_replication/follower-01/_stop?pretty' -d '{}'
+curl -XPOST -k -H 'Content-Type: application/json' -u 'admin:< Admin password >' 'https://localhost:9200/_plugins/_replication/follower-01/_stop?pretty' -d '{}'
 ```
 
 When you stop replication, the follower index un-follows the leader and becomes a standard index that you can write to. You can't restart replication after stopping it. 
 
 Get the status to confirm that the index is no longer being replicated:
 
 ```bash
-curl -XGET -k -u 'admin:admin' 'https://localhost:9200/_plugins/_replication/follower-01/_status?pretty'
+curl -XGET -k -u 'admin:< Admin password >' 'https://localhost:9200/_plugins/_replication/follower-01/_status?pretty'
```

Comment:
```suggestion
curl -XGET -k -u 'admin:<custom-admin-password>' 'https://localhost:9200/_plugins/_replication/follower-01/_status?pretty'
```

Reply:
Applied suggestion in: [cdeb90f](https://github.com/opensearch-project/documentation-website/pull/5887/commits/cdeb90ffae8be7a3ae2f9de267abc08879dd44cd)